waku/waku_core/topics/pubsub_topic.nim

```diff
@@ -67,19 +67,19 @@ proc parseStaticSharding*(
         ParsingError.invalidFormat($err)
     )
 
-  ok(NsPubsubTopic.staticSharding(clusterId, shardId))
+  ok(RelayShard.staticSharding(clusterId, shardId))
 
-proc parse*(T: type NsPubsubTopic, topic: PubsubTopic): ParsingResult[NsPubsubTopic] =
+proc parse*(T: type RelayShard, topic: PubsubTopic): ParsingResult[RelayShard] =
   ## Splits a namespaced topic string into its constituent parts.
   ## The topic string has to be in the format `/<application>/<version>/<topic-name>/<encoding>`
-  NsPubsubTopic.parseStaticSharding(topic)
+  RelayShard.parseStaticSharding(topic)
 
 # Pubsub topic compatibility
 
-converter toPubsubTopic*(topic: NsPubsubTopic): PubsubTopic =
+converter toPubsubTopic*(topic: RelayShard): PubsubTopic =
   $topic
 
-proc `==`*[T: NsPubsubTopic](x, y: T): bool =
+proc `==`*[T: RelayShard](x, y: T): bool =
   if x.clusterId != y.clusterId:
     return false
```

Comment:
That would only be partial equality. What happen is shard don't match?

Reply:
I think the other part of the proc was not properly displayed

<img width="1162" alt="image" src="https://github.com/user-attachments/assets/8ad28edf-f661-4488-b5d4-45a6ef1cd166">


I think it's ok like that right?

Reply:
Ah! NVM then.!